crates/bevy_gltf/src/vertex_attributes.rs

```diff
@@ -241,13 +241,16 @@ enum ConversionMode {
     TexCoord,
 }
 
-#[derive(Error, Debug)]
+#[derive(Error, Display, Debug)]
+#[error(ignore)]
 pub(crate) enum ConvertAttributeError {
-    #[error("Vertex attribute {0} has format {1:?} but expected {3:?} for target attribute {2}")]
+    #[display(
+        "Vertex attribute {_0} has format {_1:?} but expected {_3:?} for target attribute {_2}"
+    )]
     WrongFormat(String, VertexFormat, String, VertexFormat),
-    #[error("{0} in accessor {1}")]
+    #[display("{0} in accessor {_1}")]
     AccessFailed(AccessFailed, usize),
```

Comment:
This is not going to do what is intended in this code. This is going to print as something like "99 in accessor 99" because derive_more's expansion of this looks like `write!(formatter, "{0} in accessor {_1}", _1=self.1)` in which `{0}` refers to the `_1` named argument.

Reply:
Interesting! I'll look into that and make the appropriate fixes, I suspect this might've occurred in a few places.